grails-app/services/org/broadinstitute/orsp/NotifyService.groovy

```diff
@@ -574,13 +581,15 @@ class NotifyService implements SendgridSupport, Status {
     }
 
     def projectCGCreation(Issue issue) {
```

Comment:
I must have missed this in an earlier review - this should be returning `Map<Boolean, String>`